lib/routes/api/metadata/request.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+/**
+ * Dependencies
+ */
+
+const MAX_REDIRECTS = require('../../../../config').max_redirects;
```

Comment:
But I agree, we need a neater way of doing this. I rember seeing a solution that involved symlinking app directories into `node_modules` so that they could be required with short paths.